doc/api/http2.md

```diff
@@ -3150,13 +3150,15 @@ will result in a [`TypeError`][] being thrown.
 added: v8.4.0
 -->
 * `headers` {HTTP/2 Headers Object} An object describing the headers
-* `callback` {Function}
-
-Call [`http2stream.pushStream()`][] with the given headers, and wraps the
-given newly created [`Http2Stream`] on `Http2ServerResponse`.
+* `callback` {Function} Callback that is called once
+`http2stream.pushStream()` is finished, or when the stream is closed.
```

Comment:
hmm... the `when the stream is closed` bit is a bit confusing.

The `callback` is invoked either (a) after the pushed `Http2Stream` instead has been created and is ready for use or (b) after the attempt to create the pushed `Http2Stream` has failed or has been rejected.

Reply:
OK, fixed :)

Reply:
Well……according to what you suggested, I've included these points in my doc modifications :)